zerver/lib/streams.py

```diff
@@ -757,27 +757,57 @@ def can_access_stream_history_by_id(user_profile: UserProfile, stream_id: int) -
     return can_access_stream_history(user_profile, stream)
 
 
-def can_remove_subscribers_from_stream(
-    stream: Stream, user_profile: UserProfile, sub: Subscription | None
+def can_remove_subscribers_from_all_streams(
```

Comment:
`bulk_can_remove_subscribers_from_streams` is probably a better name -- "all_streams" sounds like it's all streams in the realm, and the plural naturally indicates. (Leaving off "bulk" makes it sound like the general capability to remove subscribers from some streams somewhere).

Reply:
[Fixed](https://github.com/zulip/zulip/compare/66bfafefd19eb711d513d5b49062567c02081742..a00f864b7fcce02c82a09c1239a9945cd8eb7b8a)